core/src/main/java/io/smallrye/openapi/runtime/scanner/spi/AnnotationScanner.java

```diff
@@ -417,9 +420,11 @@ default void createResponseFromRestMethod(final AnnotationScannerContext context
     default int getDefaultStatus(final MethodInfo method) {
         final int status;
 
-        if (method.returnType().kind() == Type.Kind.VOID) {
+        if (isVoidResponse(method)) {
             if (isPostMethod(method)) {
                 status = 201; // Created
+            } else if (isDeleteMethod(method)) {
+                status = 204; // No Content (Maybe this should be 202 Accepted ?)
```

Comment:
There is wording in the spec about this now ([link](https://github.com/eclipse/microprofile-open-api/blob/54822cd7014a4c01a781b7f281527f9c85d028c3/api/src/main/java/org/eclipse/microprofile/openapi/annotations/responses/APIResponseSchema.java#L96-L105)), I'm not sure if checking for DELETE here might have an impact. It makes sense, and maybe we can get DELETE added to the MP spec as well.

Reply:
Yea I can remove this as it will fall through to the default that is the same, but I think 202 might be a better default for delete ? I'll bring it up in the spec meeting.